packages/ui/src/bounty/modals/AddBountyModal/helpers.ts

```diff
@@ -24,11 +24,19 @@ interface Conditions {
   minWorkEntrantStake?: BalanceOf & AugmentedConst<'rxjs'>
 }
 
+export const isUrlValid = (value: string) => {
+  //check if url is valid
+  const urlValidation = value.match(
+    /^(ftp:\/\/|http:\/\/|https:\/\/)[a-z0-9]+([-.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/
+  )
+  return urlValidation !== null
```

Comment:
This regex looks ok, but it would be easier to rely on Yup for this kind of validation:
```suggestion
  return Yup.string().required().url().isValidSync(value)
```